introspection.go

```diff
@@ -157,6 +157,7 @@ type RelayItemState struct {
 	ID                      uint32 `json:"id"`
 	RemapID                 uint32 `json:"remapID"`
 	DestinationConnectionID uint32 `json:"destinationConnectionID"`
+	Tomb                    bool   `json:"tomb"`
```

Comment:
Added an option to include tombstones in state verification, in case we need it later.